db/migrations/20201207151120_create_new_storage_block_heights.sql

```diff
@@ -0,0 +1,15 @@
+-- +goose Up
+
+CREATE OR REPLACE FUNCTION api.new_storage_block_heights()
```

Comment:
wondering if we might want to name this something that makes the purpose more obvious for API users - my mind is going somewhere very verbose like `get_block_heights_for_new_untransformed_diffs()` but maybe there's a better middle ground.

Also def nbd as long as it serves the purpose for Ed

Reply:
I've gone through several names and hate them all equally. Yours is probably the "least bad".